### DISCUSSION TITLE
Using dashboards and panels as "spans" in tracing

### DISCUSSION THREAD

**Discussion by felipesere** [grafana/grafana#45358] [General]
Hello!

My colleagues and I run a Grafana instance on top of our Thanos/Prometheus instance for other teams in the company.
Often, we'll get alerts due to slow queries that originate from some dashboard/panel and its really hard to figure out which one it is.
We have `tracing` enabled for all components in our stack, but that only gives us the raw query that was sent to the Grafana backend.
Is it possible to have the Frontend component include the dashboard name and possibly panel/widget on when fetching data so the backend can attach this information to its spans? 
I'd be happy to raise a PR if someone can guide me onto the right direction.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
